timelock.props

```diff
@@ -1,11 +1,2 @@
-# Timelock server uses a higher dropwizard version so that it can have an HTTP/2 connector.
-# This is supported by Dropwizard starting v1.0.0.
-# http://www.dropwizard.io/1.0.6/docs/manual/configuration.html#http-2-over-tls
-ch.qos.logback:logback-classic = 1.1.7
-io.atomix.catalyst:catalyst-transport = 1.1.2
-io.dropwizard:* = 1.0.6
-io.dropwizard:dropwizard-jackson = 0.9.3
 io.github.openfeign:feign-core = 9.5.0
-org.eclipse.jetty:* = 9.3.9.v20160517
```

Comment:
hmm, how do the two props files interplay?

was it not sufficient to just bump to 1.3 here? and leave everything else?